docs/sources/setup/kubernetes.md

```diff
@@ -238,6 +239,100 @@ spec:
               value: "true"
 ```
 
+### Deploy Beyla unprivileged
+
+In all of the examples so far, `privileged:true` was used in the Beyla deployment `securityContext` section. While this works in all circumstances, there are ways to deploy Beyla in Kubernetes with reduced privileges, if your security configuration requires you to do so. Whether it is possible to run Beyla without `privileged:true`, depends a lot on the Kubernetes version you have and the underlying container runtime used (e.g. **Containerd**, **CRI-O** or **Docker**).
+
+The following guide is based on tests performed mainly by running `containerd` with `kubeadm`, `k3s`, `microk8s` and `kind`.
```

Comment:
Just checking, is the spelling "containerd" without an "ed" at the end correct?

Reply:
Yes, it is: https://containerd.io/